docs/examples/simulation.md

```diff
@@ -0,0 +1,392 @@
+---
+jupyter:
+  jupytext:
+    cell_metadata_filter: -all
+    formats: ipynb,md
+    text_representation:
+      extension: .md
+      format_name: markdown
+      format_version: '1.3'
+      jupytext_version: 1.13.6
+  kernelspec:
+    display_name: Python 3 (ipykernel)
+    language: python
+    name: python3
+---
+
+!!! abstract
+    This tutorial guides you through an example material synthesis workflow using the 
+    [CRIPT Python SDK](https://pypi.org/project/cript/).
+
+
+## Installation
+
+Before you start, be sure the [cript python package](https://pypi.org/project/cript/) is installed.
+
+```bash
+pip install cript
+```
+
+## Connect to CRIPT
+
+To connect to CRIPT, you must enter a `host` and an `API Token`. For most users, `host` will be `https://criptapp.org`.
+
+!!! Warning "Keep API Token Secure"
+
+    To ensure security, avoid storing sensitive information like tokens directly in your code.
+    Instead, use environment variables.
+    Storing tokens in code shared on platforms like GitHub can lead to security incidents.
+    Anyone that possesses your token can impersonate you on the [CRIPT](https://criptapp.org/) platform.
+    Consider [alternative methods for loading tokens with the CRIPT API Client]().
+    In case your token is exposed be sure to immediately generate a new token to revoke the access of the old one
+    and keep the new token safe.
+
+```python
+import cript
+
+with cript.API(host="http://development.api.mycriptapp.org/", token="123456"):
+    pass
+```
+
+You may notice, that we are not executing any code inside the context manager block.
+If you were to write a python script, compared to a jupyter notebook, you would add all the following code inside that block.
+Here in a jupyter notebook, we need to connect manually. We just have to remember to disconnect at the end.
+
+```python
+api = cript.API("http://development.api.mycriptapp.org/", None)
```

Comment:
for documentation named arguments are better because they make things clearer and make mistakes harder